roachtest: pgjdbc failed

[(roachtest).pgjdbc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2137326&tab=buildLog) on [master@229b1bba516020c1681e5e713c0c79b4efd2323e](https://github.com/cockroachdb/cockroach/commits/229b1bba516020c1681e5e713c0c79b4efd2323e):

```
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testExecuteEmptyPreparedBatch[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testPreparedNoParameters[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testEmptyClearBatch[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithAlternatingAndUnknownTypes6[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testPreparedNoParameters[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testClearBatch[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testClearPreparedEmptyBatch[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithAlternatingAndUnknownTypes4[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testStringAddBatchOnPreparedStatement[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithAlternatingAndUnknownTypes3[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithRepeatedInsertStatement[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithAlternatingAndUnknownTypes4[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testSupportsBatchUpdates[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testPreparedNoParameters[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testClearBatch[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithAlternatingAndUnknownTypes0[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testSelectInBatchThrows[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchedInsertReWriteEnabledTest.testBatchWithReWrittenBatchStatementWithSemiColon[2: autoCommit=NO, binary=REGULAR] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testClearPreparedNoArgBatch[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testWarningsAreCleared[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithAlternatingAndUnknownTypes0[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithRepeatedInsertStatement[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testPreparedStatement[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithAlternatingAndUnknownTypes6[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testPreparedStatement[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchedInsertReWriteEnabledTest.testBatchWithReWrittenSpaceAfterValues[3: autoCommit=NO, binary=FORCE] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchReturningMixedNulls[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testExecuteEmptyBatch[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithMultiInsert[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithAlternatingAndUnknownTypes2[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testServerPrepareMultipleRows[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testClearBatch[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchedInsertReWriteEnabledTest.testReWriteDisabledForPlainBatch[3: autoCommit=NO, binary=FORCE] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithAlternatingAndUnknownTypes1[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testWarningsAreCleared[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testClearPreparedEmptyBatch[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithAlternatingAndUnknownTypes2[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithMultiInsert[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testExecuteEmptyPreparedBatch[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testSelectInBatchThrows[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testStringAddBatchOnPreparedStatement[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testSupportsBatchUpdates[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testExecuteEmptyPreparedBatch[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testClearPreparedEmptyBatch[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchedInsertReWriteEnabledTest.testBatchWithReWrittenMixedCaseValues[3: autoCommit=NO, binary=FORCE] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testSupportsBatchUpdates[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithAlternatingAndUnknownTypes0[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		For a full summary look at the pgjdbc artifacts 
		
		An updated blocklist (pgjdbcBlockList20_2) is available in the artifacts' pgjdbc log

```

<details><summary>More</summary><p>


Artifacts: [/pgjdbc](https://teamcity.cockroachdb.com/viewLog.html?buildId=2137326&tab=artifacts#/pgjdbc)
Related:
- #49735 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apgjdbc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgjdbc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2154638&tab=buildLog) on [master@ef0e879cd95637ca0038b0406dc239d92b556c8e](https://github.com/cockroachdb/cockroach/commits/ef0e879cd95637ca0038b0406dc239d92b556c8e):

```
		--- FAIL: org.postgresql.test.jdbc2.optional.PoolingDataSourceTest.testJndi - unknown (unexpected)
		--- FAIL: org.postgresql.test.core.NativeQueryBindLengthTest.testBindLengthCalculation['$1$2...$9$10..$99$100'.length = 2*9+3*90+4 == 292] - unknown (unexpected)
		--- FAIL: bindInTheMiddle[194: quotes(style=', src=$o‿o$, quoted=-- $
		'$o‿o$')](org.postgresql.test.jdbc2.QuotationTest).bindInTheMiddle[194: quotes(style=', src=$o‿o$, quoted=-- $
		'$o‿o$')] - unknown (unexpected)
		--- FAIL: bindInTheMiddle[831: quotes(style=$a$, src=r, quoted=--
		/* $ */$a$r$a$)](org.postgresql.test.jdbc2.QuotationTest).bindInTheMiddle[831: quotes(style=$a$, src=r, quoted=--
		/* $ */$a$r$a$)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.QuotationTest.bindInTheMiddle[1216: quotes(style=$o‿o$, src=select?select?select?, quoted=$o‿o$select?select?select?$o‿o$)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchFailureTest.run[77: batchTest(mode=FAIL_VIA_SELECT_RUNTIME, position=ALMOST_LAST_ROW, autoCommit=NO, batchType=SIMPLE, generateKeys=NO, binary=REGULAR, insertRewrite=false)] - unknown (unexpected)
		--- FAIL: quotedString[589: quotes(style=$a$, src={extract}{extract}{extract}, quoted=/*
		$
		*//* ? *//*{fn *//* now} */$a${extract}{extract}{extract}$a$)](org.postgresql.test.jdbc2.QuotationTest).quotedString[589: quotes(style=$a$, src={extract}{extract}{extract}, quoted=/*
		$
		*//* ? *//*{fn *//* now} */$a${extract}{extract}{extract}$a$)] - unknown (unexpected)
		--- FAIL: quotedString[1153: quotes(style=$DEF$, src=y, quoted=/*
		$
		*//* ? *//*{fn *//* now} */$DEF$y$DEF$)](org.postgresql.test.jdbc2.QuotationTest).quotedString[1153: quotes(style=$DEF$, src=y, quoted=/*
		$
		*//* ? *//*{fn *//* now} */$DEF$y$DEF$)] - unknown (unexpected)
		--- FAIL: bindInTheMiddle[1350: quotes(style=$o‿o$, src=$DEF$$DEF$$DEF$, quoted=-- $
		$o‿o$$DEF$$DEF$$DEF$$o‿o$)](org.postgresql.test.jdbc2.QuotationTest).bindInTheMiddle[1350: quotes(style=$o‿o$, src=$DEF$$DEF$$DEF$, quoted=-- $
		$o‿o$$DEF$$DEF$$DEF$$o‿o$)] - unknown (unexpected)
		For a full summary look at the pgjdbc artifacts 
		
		An updated blocklist (pgjdbcBlockList20_2) is available in the artifacts' pgjdbc log

	cluster.go:1571,context.go:135,cluster.go:1560,test_runner.go:823: dead node detection: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod monitor teamcity-2154638-1596695735-59-n1cpu4 --oneshot --ignore-empty-nodes: exit status 1 1: dead
		Error: UNCLASSIFIED_PROBLEM: 1: dead
		(1) UNCLASSIFIED_PROBLEM
		Wraps: (2) attached stack trace
		  | main.glob..func13
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1115
		  | main.wrap.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:266
		  | github.com/spf13/cobra.(*Command).execute
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra/command.go:830
		  | github.com/spf13/cobra.(*Command).ExecuteC
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra/command.go:914
		  | github.com/spf13/cobra.(*Command).Execute
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/vendor/github.com/spf13/cobra/command.go:864
		  | main.main
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1808
		  | runtime.main
		  | 	/usr/local/go/src/runtime/proc.go:203
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1373
		Wraps: (3) 3 safe details enclosed
		Wraps: (4) 1: dead
		Error types: (1) errors.Unclassified (2) *withstack.withStack (3) *safedetails.withSafeDetails (4) *errors.errorString

```

<details><summary>More</summary><p>


Artifacts: [/pgjdbc](https://teamcity.cockroachdb.com/viewLog.html?buildId=2154638&tab=artifacts#/pgjdbc)
Related:
- #49735 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apgjdbc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgjdbc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2159276&tab=buildLog) on [master@d310eb8eb6da5df4ac4407a2c90d898acdfdddf3](https://github.com/cockroachdb/cockroach/commits/d310eb8eb6da5df4ac4407a2c90d898acdfdddf3):

```
		--- FAIL: org.postgresql.test.jdbc4.XmlTest.testDOMParse - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[499: autorollback(autoSave=ALWAYS, cleanSavePoint=FALSE, autoCommit=YES, failMode=SELECT, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[771: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=TRUE, autoCommit=NO, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=true, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[928: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=FALSE, autoCommit=NO, failMode=DEALLOCATE, continueMode=COMMIT, flushOnDeallocate=false, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc42.SetObject310Test.testSetLocalTimeAndReadBack[binary = FORCE] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[437: autorollback(autoSave=ALWAYS, cleanSavePoint=TRUE, autoCommit=NO, failMode=DEALLOCATE, continueMode=COMMIT, flushOnDeallocate=false, hastransaction=false, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[501: autorollback(autoSave=ALWAYS, cleanSavePoint=FALSE, autoCommit=YES, failMode=SELECT, continueMode=SELECT, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[548: autorollback(autoSave=ALWAYS, cleanSavePoint=FALSE, autoCommit=YES, failMode=DISCARD, continueMode=SELECT, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc3.GeneratedKeysTest.testWithInsertSelect[returningInQuery = AB, binary = FORCE] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[78: autorollback(autoSave=NEVER, cleanSavePoint=TRUE, autoCommit=NO, failMode=SELECT, continueMode=COMMIT, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[108: autorollback(autoSave=NEVER, cleanSavePoint=TRUE, autoCommit=NO, failMode=DEALLOCATE, continueMode=COMMIT, flushOnDeallocate=false, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[573: autorollback(autoSave=ALWAYS, cleanSavePoint=FALSE, autoCommit=NO, failMode=SELECT, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=true, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[684: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=TRUE, autoCommit=YES, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=false, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithRepeatedInsertStatement[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc3.GeneratedKeysTest.testReturningWithTrailingSemicolon[returningInQuery = STAR, binary = FORCE] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc42.SetObject310Test.testTimeStampRounding[binary = FORCE] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc3.Jdbc3SavepointTest.testAutoCommitFails - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[906: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=FALSE, autoCommit=NO, failMode=SELECT, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc42.SetObject310Test.testSetLocalDateWithoutType[binary = FORCE] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[712: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=TRUE, autoCommit=YES, failMode=DISCARD, continueMode=SELECT, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc3.GeneratedKeysTest.selectWithGeneratedKeysViaPreparedExecuteQuery[returningInQuery = STAR, binary = REGULAR] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[518: autorollback(autoSave=ALWAYS, cleanSavePoint=FALSE, autoCommit=YES, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[708: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=TRUE, autoCommit=YES, failMode=DISCARD, continueMode=IS_VALID, flushOnDeallocate=false, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		For a full summary look at the pgjdbc artifacts 
		
		An updated blocklist (pgjdbcBlockList20_2) is available in the artifacts' pgjdbc log

	cluster.go:1571,context.go:135,cluster.go:1560,test_runner.go:823: dead node detection: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod monitor teamcity-2159276-1596781348-63-n1cpu4 --oneshot --ignore-empty-nodes: exit status 1 1: dead
		Error: UNCLASSIFIED_PROBLEM: 1: dead
		(1) UNCLASSIFIED_PROBLEM
		Wraps: (2) attached stack trace
		  | main.glob..func13
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1115
		  | main.wrap.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:266
		  | github.com/spf13/cobra.(*Command).execute
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:830
		  | github.com/spf13/cobra.(*Command).ExecuteC
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914
		  | github.com/spf13/cobra.(*Command).Execute
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
		  | main.main
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1808
		  | runtime.main
		  | 	/usr/local/go/src/runtime/proc.go:203
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1373
		Wraps: (3) 3 safe details enclosed
		Wraps: (4) 1: dead
		Error types: (1) errors.Unclassified (2) *withstack.withStack (3) *safedetails.withSafeDetails (4) *errors.errorString

```

<details><summary>More</summary><p>


Artifacts: [/pgjdbc](https://teamcity.cockroachdb.com/viewLog.html?buildId=2159276&tab=artifacts#/pgjdbc)
Related:
- #52506 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #49735 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apgjdbc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgjdbc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2163421&tab=buildLog) on [master@cf44e2e11363a43de2c13987378a3fe50b460e08](https://github.com/cockroachdb/cockroach/commits/cf44e2e11363a43de2c13987378a3fe50b460e08):

```
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[867: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=FALSE, autoCommit=YES, failMode=DISCARD, continueMode=COMMIT, flushOnDeallocate=false, hastransaction=false, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[938: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=FALSE, autoCommit=NO, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[929: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=FALSE, autoCommit=NO, failMode=DEALLOCATE, continueMode=COMMIT, flushOnDeallocate=false, hastransaction=false, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[250: autorollback(autoSave=NEVER, cleanSavePoint=FALSE, autoCommit=NO, failMode=SELECT, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[519: autorollback(autoSave=ALWAYS, cleanSavePoint=FALSE, autoCommit=YES, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc3.GeneratedKeysTest.testWithInsertInsert[returningInQuery = NO, binary = FORCE] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[127: autorollback(autoSave=NEVER, cleanSavePoint=TRUE, autoCommit=NO, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=false, hastransaction=false, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc42.GetObject310Test.testGetTimestampWithTimeZone[binary = REGULAR] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc4.XmlTest.testSAXWrite - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc3.GeneratedKeysTest.testStatementUpdateCount[returningInQuery = NO, binary = REGULAR] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[293: autorollback(autoSave=NEVER, cleanSavePoint=FALSE, autoCommit=NO, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=true, hastransaction=true, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[748: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=TRUE, autoCommit=NO, failMode=SELECT, continueMode=SELECT, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc42.SetObject310Test.testSetLocalDateWithoutType[binary = FORCE] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[201: autorollback(autoSave=NEVER, cleanSavePoint=FALSE, autoCommit=YES, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=false, hastransaction=false, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc3.GeneratedKeysTest.testMultipleRows[returningInQuery = AB, binary = REGULAR] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc3.GeneratedKeysTest.selectWithGeneratedKeysViaPreparedExecuteQuery[returningInQuery = AB, binary = FORCE] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc3.GeneratedKeysTest.testPSUpdate[returningInQuery = AB, binary = FORCE] - unknown (unexpected)
		--- FAIL: org.postgresql.test.extensions.HStoreTest.testHStoreSendEscaped - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc3.GeneratedKeysTest.testGeneratedKeys[returningInQuery = STAR, binary = REGULAR] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc3.GeneratedKeysTest.breakDescribeOnFirstServerPreparedExecution[returningInQuery = NO, binary = REGULAR] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc4.ClientInfoTest.testExplicitSetAppNameNotificationIsParsed - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[76: autorollback(autoSave=NEVER, cleanSavePoint=TRUE, autoCommit=NO, failMode=SELECT, continueMode=COMMIT, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[541: autorollback(autoSave=ALWAYS, cleanSavePoint=FALSE, autoCommit=YES, failMode=DISCARD, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=STAR)] - unknown (unexpected)
		For a full summary look at the pgjdbc artifacts 
		
		An updated blocklist (pgjdbcBlockList20_2) is available in the artifacts' pgjdbc log

	cluster.go:1571,context.go:135,cluster.go:1560,test_runner.go:823: dead node detection: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod monitor teamcity-2163421-1596954166-09-n1cpu4 --oneshot --ignore-empty-nodes: exit status 1 1: dead
		Error: UNCLASSIFIED_PROBLEM: 1: dead
		(1) UNCLASSIFIED_PROBLEM
		Wraps: (2) attached stack trace
		  | main.glob..func13
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1115
		  | main.wrap.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:266
		  | github.com/spf13/cobra.(*Command).execute
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:830
		  | github.com/spf13/cobra.(*Command).ExecuteC
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914
		  | github.com/spf13/cobra.(*Command).Execute
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
		  | main.main
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1808
		  | runtime.main
		  | 	/usr/local/go/src/runtime/proc.go:203
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1357
		Wraps: (3) 3 safe details enclosed
		Wraps: (4) 1: dead
		Error types: (1) errors.Unclassified (2) *withstack.withStack (3) *safedetails.withSafeDetails (4) *errors.errorString

```

<details><summary>More</summary><p>


Artifacts: [/pgjdbc](https://teamcity.cockroachdb.com/viewLog.html?buildId=2163421&tab=artifacts#/pgjdbc)
Related:
- #52506 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #49735 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apgjdbc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgjdbc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2164877&tab=buildLog) on [master@fdf54ebc69c7d975f8f1dfbf28ced79ddeb83ee4](https://github.com/cockroachdb/cockroach/commits/fdf54ebc69c7d975f8f1dfbf28ced79ddeb83ee4):

```
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[359: autorollback(autoSave=ALWAYS, cleanSavePoint=TRUE, autoCommit=YES, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=false, hastransaction=false, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[209: autorollback(autoSave=NEVER, cleanSavePoint=FALSE, autoCommit=YES, failMode=DISCARD, continueMode=COMMIT, flushOnDeallocate=false, hastransaction=false, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[345: autorollback(autoSave=ALWAYS, cleanSavePoint=TRUE, autoCommit=YES, failMode=DEALLOCATE, continueMode=COMMIT, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[544: autorollback(autoSave=ALWAYS, cleanSavePoint=FALSE, autoCommit=YES, failMode=DISCARD, continueMode=IS_VALID, flushOnDeallocate=false, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[582: autorollback(autoSave=ALWAYS, cleanSavePoint=FALSE, autoCommit=NO, failMode=SELECT, continueMode=SELECT, flushOnDeallocate=true, hastransaction=true, sql=WITH_INSERT_SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc3.GeneratedKeysTest.testUpdate[returningInQuery = A, binary = FORCE] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc3.GeneratedKeysTest.selectWithGeneratedKeysViaPreparedExecute[returningInQuery = A, binary = FORCE] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[225: autorollback(autoSave=NEVER, cleanSavePoint=FALSE, autoCommit=YES, failMode=DISCARD, continueMode=SELECT, flushOnDeallocate=false, hastransaction=false, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc3.GeneratedKeysTest.selectWithGeneratedKeysViaNonPrepared[returningInQuery = NO, binary = FORCE] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[865: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=FALSE, autoCommit=YES, failMode=DISCARD, continueMode=COMMIT, flushOnDeallocate=false, hastransaction=false, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[175: autorollback(autoSave=NEVER, cleanSavePoint=FALSE, autoCommit=YES, failMode=SELECT, continueMode=SELECT, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[443: autorollback(autoSave=ALWAYS, cleanSavePoint=TRUE, autoCommit=NO, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=true, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc3.GeneratedKeysTest.selectWithGeneratedKeysViaNonPrepared[returningInQuery = STAR, binary = FORCE] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[137: autorollback(autoSave=NEVER, cleanSavePoint=TRUE, autoCommit=NO, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=false, hastransaction=true, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[783: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=TRUE, autoCommit=NO, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=false, hastransaction=false, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc3.GeneratedKeysTest.selectWithGeneratedKeysViaPreparedExecute[returningInQuery = AB, binary = FORCE] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[760: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=TRUE, autoCommit=NO, failMode=DEALLOCATE, continueMode=COMMIT, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc4.CharacterStreamTest.testKnownLongLength10Kb - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[499: autorollback(autoSave=ALWAYS, cleanSavePoint=FALSE, autoCommit=YES, failMode=SELECT, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[673: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=TRUE, autoCommit=YES, failMode=DEALLOCATE, continueMode=COMMIT, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[601: autorollback(autoSave=ALWAYS, cleanSavePoint=FALSE, autoCommit=NO, failMode=DEALLOCATE, continueMode=COMMIT, flushOnDeallocate=false, hastransaction=false, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc4.WrapperTest.testStatementIsWrapperForPrivate - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[60: autorollback(autoSave=NEVER, cleanSavePoint=TRUE, autoCommit=YES, failMode=DISCARD, continueMode=SELECT, flushOnDeallocate=false, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		For a full summary look at the pgjdbc artifacts 
		
		An updated blocklist (pgjdbcBlockList20_2) is available in the artifacts' pgjdbc log

	cluster.go:1571,context.go:135,cluster.go:1560,test_runner.go:823: dead node detection: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod monitor teamcity-2164877-1597040336-16-n1cpu4 --oneshot --ignore-empty-nodes: exit status 1 1: dead
		Error: UNCLASSIFIED_PROBLEM: 1: dead
		(1) UNCLASSIFIED_PROBLEM
		Wraps: (2) attached stack trace
		  | main.glob..func13
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1115
		  | main.wrap.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:266
		  | github.com/spf13/cobra.(*Command).execute
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:830
		  | github.com/spf13/cobra.(*Command).ExecuteC
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914
		  | github.com/spf13/cobra.(*Command).Execute
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
		  | main.main
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1808
		  | runtime.main
		  | 	/usr/local/go/src/runtime/proc.go:203
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1357
		Wraps: (3) 3 safe details enclosed
		Wraps: (4) 1: dead
		Error types: (1) errors.Unclassified (2) *withstack.withStack (3) *safedetails.withSafeDetails (4) *errors.errorString

```

<details><summary>More</summary><p>


Artifacts: [/pgjdbc](https://teamcity.cockroachdb.com/viewLog.html?buildId=2164877&tab=artifacts#/pgjdbc)
Related:
- #52506 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #49735 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apgjdbc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgjdbc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2168253&tab=buildLog) on [master@b7cf171ae2d77f3adababe7f42b2cfefc795a2ba](https://github.com/cockroachdb/cockroach/commits/b7cf171ae2d77f3adababe7f42b2cfefc795a2ba):

```
		--- FAIL: org.postgresql.test.jdbc4.WrapperTest.testConnectionIsWrapperForPrivate - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc4.BinaryStreamTest.testUnknownLength200Kb - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[738: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=TRUE, autoCommit=NO, failMode=SELECT, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=true, sql=WITH_INSERT_SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[741: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=TRUE, autoCommit=NO, failMode=SELECT, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[735: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=TRUE, autoCommit=NO, failMode=SELECT, continueMode=COMMIT, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc4.WrapperTest.testStatementIsWrapperForPGStatement - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[359: autorollback(autoSave=ALWAYS, cleanSavePoint=TRUE, autoCommit=YES, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=false, hastransaction=false, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[868: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=FALSE, autoCommit=YES, failMode=DISCARD, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[305: autorollback(autoSave=NEVER, cleanSavePoint=FALSE, autoCommit=NO, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=false, hastransaction=false, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[717: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=TRUE, autoCommit=YES, failMode=DISCARD, continueMode=SELECT, flushOnDeallocate=false, hastransaction=false, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[494: autorollback(autoSave=ALWAYS, cleanSavePoint=FALSE, autoCommit=YES, failMode=SELECT, continueMode=COMMIT, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc4.UUIDTest.testUUIDString[binary=REGULAR, stringType=UNSPECIFIED] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[527: autorollback(autoSave=ALWAYS, cleanSavePoint=FALSE, autoCommit=YES, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc4.CharacterStreamTest.testKnownLongLengthEmpty - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[879: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=FALSE, autoCommit=YES, failMode=DISCARD, continueMode=SELECT, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[748: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=TRUE, autoCommit=NO, failMode=SELECT, continueMode=SELECT, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[553: autorollback(autoSave=ALWAYS, cleanSavePoint=FALSE, autoCommit=YES, failMode=DISCARD, continueMode=SELECT, flushOnDeallocate=false, hastransaction=false, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[602: autorollback(autoSave=ALWAYS, cleanSavePoint=FALSE, autoCommit=NO, failMode=DEALLOCATE, continueMode=COMMIT, flushOnDeallocate=false, hastransaction=false, sql=WITH_INSERT_SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[31: autorollback(autoSave=NEVER, cleanSavePoint=TRUE, autoCommit=YES, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=false, hastransaction=false, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[677: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=TRUE, autoCommit=YES, failMode=DEALLOCATE, continueMode=COMMIT, flushOnDeallocate=false, hastransaction=false, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[221: autorollback(autoSave=NEVER, cleanSavePoint=FALSE, autoCommit=YES, failMode=DISCARD, continueMode=SELECT, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[333: autorollback(autoSave=ALWAYS, cleanSavePoint=TRUE, autoCommit=YES, failMode=SELECT, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[185: autorollback(autoSave=NEVER, cleanSavePoint=FALSE, autoCommit=YES, failMode=DEALLOCATE, continueMode=COMMIT, flushOnDeallocate=false, hastransaction=false, sql=SELECT, columns=STAR)] - unknown (unexpected)
		For a full summary look at the pgjdbc artifacts 
		
		An updated blocklist (pgjdbcBlockList20_2) is available in the artifacts' pgjdbc log

	cluster.go:1571,context.go:135,cluster.go:1560,test_runner.go:823: dead node detection: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod monitor teamcity-2168253-1597128027-15-n1cpu4 --oneshot --ignore-empty-nodes: exit status 1 1: dead
		Error: UNCLASSIFIED_PROBLEM: 1: dead
		(1) UNCLASSIFIED_PROBLEM
		Wraps: (2) attached stack trace
		  | main.glob..func13
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1115
		  | main.wrap.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:266
		  | github.com/spf13/cobra.(*Command).execute
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:830
		  | github.com/spf13/cobra.(*Command).ExecuteC
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914
		  | github.com/spf13/cobra.(*Command).Execute
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
		  | main.main
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1808
		  | runtime.main
		  | 	/usr/local/go/src/runtime/proc.go:203
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1357
		Wraps: (3) 3 safe details enclosed
		Wraps: (4) 1: dead
		Error types: (1) errors.Unclassified (2) *withstack.withStack (3) *safedetails.withSafeDetails (4) *errors.errorString

```

<details><summary>More</summary><p>


Artifacts: [/pgjdbc](https://teamcity.cockroachdb.com/viewLog.html?buildId=2168253&tab=artifacts#/pgjdbc)
Related:
- #52506 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #49735 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apgjdbc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgjdbc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2177056&tab=buildLog) on [master@457f1af5d19e0bcef9b5ea9e9125b3332717a86c](https://github.com/cockroachdb/cockroach/commits/457f1af5d19e0bcef9b5ea9e9125b3332717a86c):

```
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[246: autorollback(autoSave=NEVER, cleanSavePoint=FALSE, autoCommit=NO, failMode=SELECT, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=true, sql=WITH_INSERT_SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[104: autorollback(autoSave=NEVER, cleanSavePoint=TRUE, autoCommit=NO, failMode=DEALLOCATE, continueMode=COMMIT, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[272: autorollback(autoSave=NEVER, cleanSavePoint=FALSE, autoCommit=NO, failMode=DEALLOCATE, continueMode=COMMIT, flushOnDeallocate=false, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[795: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=TRUE, autoCommit=NO, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=false, hastransaction=true, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testStringAddBatchOnPreparedStatement[binary = FORCE, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testBatchWithRepeatedInsertStatement[binary = REGULAR, insertRewrite = false] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc42.GetObject310Test.testGetLocalTime[binary = FORCE] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[581: autorollback(autoSave=ALWAYS, cleanSavePoint=FALSE, autoCommit=NO, failMode=SELECT, continueMode=SELECT, flushOnDeallocate=true, hastransaction=true, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[871: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=FALSE, autoCommit=YES, failMode=DISCARD, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[363: autorollback(autoSave=ALWAYS, cleanSavePoint=TRUE, autoCommit=YES, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[34: autorollback(autoSave=NEVER, cleanSavePoint=TRUE, autoCommit=YES, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc42.SetObject310Test.testSetLocalDateTimeBc[binary = REGULAR] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[282: autorollback(autoSave=NEVER, cleanSavePoint=FALSE, autoCommit=NO, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[789: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=TRUE, autoCommit=NO, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[226: autorollback(autoSave=NEVER, cleanSavePoint=FALSE, autoCommit=YES, failMode=DISCARD, continueMode=SELECT, flushOnDeallocate=false, hastransaction=false, sql=WITH_INSERT_SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testPreparedStatement[binary = FORCE, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[376: autorollback(autoSave=ALWAYS, cleanSavePoint=TRUE, autoCommit=YES, failMode=DISCARD, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[198: autorollback(autoSave=NEVER, cleanSavePoint=FALSE, autoCommit=YES, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[123: autorollback(autoSave=NEVER, cleanSavePoint=TRUE, autoCommit=NO, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=false, hastransaction=true, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[342: autorollback(autoSave=ALWAYS, cleanSavePoint=TRUE, autoCommit=YES, failMode=ALTER, continueMode=SELECT, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.BatchExecuteTest.testStringAddBatchOnPreparedStatement[binary = REGULAR, insertRewrite = true] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[628: autorollback(autoSave=ALWAYS, cleanSavePoint=FALSE, autoCommit=NO, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=false, hastransaction=true, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[701: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=TRUE, autoCommit=YES, failMode=DISCARD, continueMode=COMMIT, flushOnDeallocate=false, hastransaction=false, sql=SELECT, columns=STAR)] - unknown (unexpected)
		For a full summary look at the pgjdbc artifacts 
		
		An updated blocklist (pgjdbcBlockList20_2) is available in the artifacts' pgjdbc log

	cluster.go:1612,context.go:135,cluster.go:1601,test_runner.go:823: dead node detection: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod monitor teamcity-2177056-1597299797-26-n1cpu4 --oneshot --ignore-empty-nodes: exit status 1 1: dead
		Error: UNCLASSIFIED_PROBLEM: 1: dead
		(1) UNCLASSIFIED_PROBLEM
		Wraps: (2) attached stack trace
		  | main.glob..func13
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1115
		  | main.wrap.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:266
		  | github.com/spf13/cobra.(*Command).execute
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:830
		  | github.com/spf13/cobra.(*Command).ExecuteC
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914
		  | github.com/spf13/cobra.(*Command).Execute
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
		  | main.main
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1808
		  | runtime.main
		  | 	/usr/local/go/src/runtime/proc.go:203
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1357
		Wraps: (3) 3 safe details enclosed
		Wraps: (4) 1: dead
		Error types: (1) errors.Unclassified (2) *withstack.withStack (3) *safedetails.withSafeDetails (4) *errors.errorString

```

<details><summary>More</summary><p>


Artifacts: [/pgjdbc](https://teamcity.cockroachdb.com/viewLog.html?buildId=2177056&tab=artifacts#/pgjdbc)
Related:
- #52506 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #49735 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apgjdbc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgjdbc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2180641&tab=buildLog) on [master@1049eaa26d983a0449049264d6c2ee2cc27454e2](https://github.com/cockroachdb/cockroach/commits/1049eaa26d983a0449049264d6c2ee2cc27454e2):

```
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[699: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=TRUE, autoCommit=YES, failMode=DISCARD, continueMode=COMMIT, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[772: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=TRUE, autoCommit=NO, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[412: autorollback(autoSave=ALWAYS, cleanSavePoint=TRUE, autoCommit=NO, failMode=SELECT, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[418: autorollback(autoSave=ALWAYS, cleanSavePoint=TRUE, autoCommit=NO, failMode=SELECT, continueMode=SELECT, flushOnDeallocate=true, hastransaction=true, sql=WITH_INSERT_SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[777: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=TRUE, autoCommit=NO, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=false, hastransaction=true, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[414: autorollback(autoSave=ALWAYS, cleanSavePoint=TRUE, autoCommit=NO, failMode=SELECT, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[283: autorollback(autoSave=NEVER, cleanSavePoint=FALSE, autoCommit=NO, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[303: autorollback(autoSave=NEVER, cleanSavePoint=FALSE, autoCommit=NO, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=false, hastransaction=true, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[360: autorollback(autoSave=ALWAYS, cleanSavePoint=TRUE, autoCommit=YES, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[134: autorollback(autoSave=NEVER, cleanSavePoint=TRUE, autoCommit=NO, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[126: autorollback(autoSave=NEVER, cleanSavePoint=TRUE, autoCommit=NO, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=false, hastransaction=false, sql=WITH_INSERT_SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[63: autorollback(autoSave=NEVER, cleanSavePoint=TRUE, autoCommit=YES, failMode=DISCARD, continueMode=SELECT, flushOnDeallocate=false, hastransaction=false, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[272: autorollback(autoSave=NEVER, cleanSavePoint=FALSE, autoCommit=NO, failMode=DEALLOCATE, continueMode=COMMIT, flushOnDeallocate=false, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- PASS: org.postgresql.test.jdbc42.LargeCountJdbc42Test.testExecuteLargePreparedStatementStatementLoopSMALL[binary = FORCE, insertRewrite = false] - https://github.com/cockroachdb/cockroach/issues/43352 (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[223: autorollback(autoSave=NEVER, cleanSavePoint=FALSE, autoCommit=YES, failMode=DISCARD, continueMode=SELECT, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[62: autorollback(autoSave=NEVER, cleanSavePoint=TRUE, autoCommit=YES, failMode=DISCARD, continueMode=SELECT, flushOnDeallocate=false, hastransaction=false, sql=WITH_INSERT_SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[129: autorollback(autoSave=NEVER, cleanSavePoint=TRUE, autoCommit=NO, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=true, hastransaction=true, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[871: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=FALSE, autoCommit=YES, failMode=DISCARD, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[78: autorollback(autoSave=NEVER, cleanSavePoint=TRUE, autoCommit=NO, failMode=SELECT, continueMode=COMMIT, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[45: autorollback(autoSave=NEVER, cleanSavePoint=TRUE, autoCommit=YES, failMode=DISCARD, continueMode=COMMIT, flushOnDeallocate=false, hastransaction=false, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[124: autorollback(autoSave=NEVER, cleanSavePoint=TRUE, autoCommit=NO, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=false, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[783: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=TRUE, autoCommit=NO, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=false, hastransaction=false, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[194: autorollback(autoSave=NEVER, cleanSavePoint=FALSE, autoCommit=YES, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=false, hastransaction=false, sql=WITH_INSERT_SELECT, columns=EXACT)] - unknown (unexpected)
		For a full summary look at the pgjdbc artifacts 
		
		An updated blocklist (pgjdbcBlockList20_2) is available in the artifacts' pgjdbc log

	cluster.go:1612,context.go:135,cluster.go:1601,test_runner.go:823: dead node detection: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod monitor teamcity-2180641-1597388747-69-n1cpu4 --oneshot --ignore-empty-nodes: exit status 1 1: dead
		Error: UNCLASSIFIED_PROBLEM: 1: dead
		(1) UNCLASSIFIED_PROBLEM
		Wraps: (2) attached stack trace
		  | main.glob..func13
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1115
		  | main.wrap.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:266
		  | github.com/spf13/cobra.(*Command).execute
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:830
		  | github.com/spf13/cobra.(*Command).ExecuteC
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914
		  | github.com/spf13/cobra.(*Command).Execute
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
		  | main.main
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1808
		  | runtime.main
		  | 	/usr/local/go/src/runtime/proc.go:203
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1357
		Wraps: (3) 3 safe details enclosed
		Wraps: (4) 1: dead
		Error types: (1) errors.Unclassified (2) *withstack.withStack (3) *safedetails.withSafeDetails (4) *errors.errorString

```

<details><summary>More</summary><p>


Artifacts: [/pgjdbc](https://teamcity.cockroachdb.com/viewLog.html?buildId=2180641&tab=artifacts#/pgjdbc)
Related:
- #52506 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #49735 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apgjdbc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).pgjdbc failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2184311&tab=buildLog) on [master@d106b79cb3838bbc1fb163e8d8216488130b521e](https://github.com/cockroachdb/cockroach/commits/d106b79cb3838bbc1fb163e8d8216488130b521e):

```
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[360: autorollback(autoSave=ALWAYS, cleanSavePoint=TRUE, autoCommit=YES, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[412: autorollback(autoSave=ALWAYS, cleanSavePoint=TRUE, autoCommit=NO, failMode=SELECT, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[87: autorollback(autoSave=NEVER, cleanSavePoint=TRUE, autoCommit=NO, failMode=SELECT, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		--- PASS: org.postgresql.test.jdbc42.LargeCountJdbc42Test.testExecuteLargeUpdateStatementSELECT[binary = FORCE, insertRewrite = true] - https://github.com/cockroachdb/cockroach/issues/43352 (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[542: autorollback(autoSave=ALWAYS, cleanSavePoint=FALSE, autoCommit=YES, failMode=DISCARD, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[847: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=FALSE, autoCommit=YES, failMode=DEALLOCATE, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[668: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=TRUE, autoCommit=YES, failMode=ALTER, continueMode=SELECT, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[924: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=FALSE, autoCommit=NO, failMode=DEALLOCATE, continueMode=COMMIT, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[665: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=TRUE, autoCommit=YES, failMode=SELECT, continueMode=SELECT, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[737: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=TRUE, autoCommit=NO, failMode=SELECT, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=true, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[257: autorollback(autoSave=NEVER, cleanSavePoint=FALSE, autoCommit=NO, failMode=SELECT, continueMode=SELECT, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- PASS: org.postgresql.test.jdbc42.LargeCountJdbc42Test.testNullExecuteLargeBatchPreparedStatement[binary = REGULAR, insertRewrite = true] - https://github.com/cockroachdb/cockroach/issues/43352 (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[48: autorollback(autoSave=NEVER, cleanSavePoint=TRUE, autoCommit=YES, failMode=DISCARD, continueMode=IS_VALID, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[461: autorollback(autoSave=ALWAYS, cleanSavePoint=TRUE, autoCommit=NO, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=STAR)] - unknown (unexpected)
		--- PASS: org.postgresql.test.jdbc42.LargeCountJdbc42Test.testGetLargeUpdateCountStatementSMALL[binary = FORCE, insertRewrite = true] - https://github.com/cockroachdb/cockroach/issues/43352 (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[432: autorollback(autoSave=ALWAYS, cleanSavePoint=TRUE, autoCommit=NO, failMode=DEALLOCATE, continueMode=COMMIT, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[242: autorollback(autoSave=NEVER, cleanSavePoint=FALSE, autoCommit=NO, failMode=SELECT, continueMode=COMMIT, flushOnDeallocate=true, hastransaction=false, sql=WITH_INSERT_SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[712: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=TRUE, autoCommit=YES, failMode=DISCARD, continueMode=SELECT, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[40: autorollback(autoSave=NEVER, cleanSavePoint=TRUE, autoCommit=YES, failMode=DISCARD, continueMode=COMMIT, flushOnDeallocate=true, hastransaction=false, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[744: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=TRUE, autoCommit=NO, failMode=SELECT, continueMode=SELECT, flushOnDeallocate=true, hastransaction=true, sql=SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[366: autorollback(autoSave=ALWAYS, cleanSavePoint=TRUE, autoCommit=YES, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=false, hastransaction=false, sql=WITH_INSERT_SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[718: autorollback(autoSave=CONSERVATIVE, cleanSavePoint=TRUE, autoCommit=YES, failMode=DISCARD, continueMode=SELECT, flushOnDeallocate=false, hastransaction=false, sql=WITH_INSERT_SELECT, columns=EXACT)] - unknown (unexpected)
		--- FAIL: org.postgresql.test.jdbc2.AutoRollbackTestSuite.run[531: autorollback(autoSave=ALWAYS, cleanSavePoint=FALSE, autoCommit=YES, failMode=DEALLOCATE, continueMode=SELECT, flushOnDeallocate=false, hastransaction=false, sql=WITH_INSERT_SELECT, columns=STAR)] - unknown (unexpected)
		For a full summary look at the pgjdbc artifacts 
		
		An updated blocklist (pgjdbcBlockList20_2) is available in the artifacts' pgjdbc log

	cluster.go:1612,context.go:135,cluster.go:1601,test_runner.go:823: dead node detection: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/bin/roachprod monitor teamcity-2184311-1597472915-17-n1cpu4 --oneshot --ignore-empty-nodes: exit status 1 1: dead
		Error: UNCLASSIFIED_PROBLEM: 1: dead
		(1) UNCLASSIFIED_PROBLEM
		Wraps: (2) attached stack trace
		  | main.glob..func13
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1115
		  | main.wrap.func1
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:266
		  | github.com/spf13/cobra.(*Command).execute
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:830
		  | github.com/spf13/cobra.(*Command).ExecuteC
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:914
		  | github.com/spf13/cobra.(*Command).Execute
		  | 	/home/agent/work/.go/pkg/mod/github.com/spf13/cobra@v0.0.5/command.go:864
		  | main.main
		  | 	/home/agent/work/.go/src/github.com/cockroachdb/cockroach/pkg/cmd/roachprod/main.go:1808
		  | runtime.main
		  | 	/usr/local/go/src/runtime/proc.go:203
		  | runtime.goexit
		  | 	/usr/local/go/src/runtime/asm_amd64.s:1357
		Wraps: (3) 3 safe details enclosed
		Wraps: (4) 1: dead
		Error types: (1) errors.Unclassified (2) *withstack.withStack (3) *safedetails.withSafeDetails (4) *errors.errorString

```

<details><summary>More</summary><p>


Artifacts: [/pgjdbc](https://teamcity.cockroachdb.com/viewLog.html?buildId=2184311&tab=artifacts#/pgjdbc)
Related:
- #52506 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-19.2](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-19.2) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

- #49735 roachtest: pgjdbc failed  [C-test-failure](https://api.github.com/repos/cockroachdb/cockroach/labels/C-test-failure) [O-roachtest](https://api.github.com/repos/cockroachdb/cockroach/labels/O-roachtest) [O-robot](https://api.github.com/repos/cockroachdb/cockroach/labels/O-robot) [branch-release-20.1](https://api.github.com/repos/cockroachdb/cockroach/labels/branch-release-20.1) [release-blocker](https://api.github.com/repos/cockroachdb/cockroach/labels/release-blocker)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Apgjdbc.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/sql/conn_executor_exec.go
+++ b/pkg/sql/conn_executor_exec.go
@@ -181,7 +181,11 @@ func (ex *connExecutor) execPortal(
 			// Note that the portal is considered exhausted regardless of
 			// the fact whether an error occurred or not - if it did, we
 			// still don't want to re-execute the portal from scratch.
-			ex.exhaustPortal(portalName)
+			// The current statement may have just closed and deleted the portal,
+			// so only exhaust it if it still exists.
+			if _, ok := ex.extraTxnState.prepStmtsNamespace.portals[portalName]; ok {
+				ex.exhaustPortal(portalName)
+			}
 		}
 	default:
 		ev, payload, err = ex.execStmt(stmtCtx, curStmt, stmtRes, pinfo)

--- a/pkg/sql/pgwire/testdata/pgtest/pgjdbc
+++ b/pkg/sql/pgwire/testdata/pgtest/pgjdbc
@@ -1,3 +1,55 @@
+# deallocate_test checks that we can run DEALLOCATE ALL using a prepared
+# statement. See #52915.
+send
+Query {"String": "DROP TABLE IF EXISTS deallocate_test"}
+----
+
+until ignore=NoticeResponse
+ReadyForQuery
+----
+{"Type":"CommandComplete","CommandTag":"DROP TABLE"}
+{"Type":"ReadyForQuery","TxStatus":"I"}
+
+send
+Query {"String": "CREATE TABLE deallocate_test (a INT)"}
+----
+
+until
+ReadyForQuery
+----
+{"Type":"CommandComplete","CommandTag":"CREATE TABLE"}
+{"Type":"ReadyForQuery","TxStatus":"I"}
+
+# 80 = ASCII 'P' for Portal
+send
+Parse {"Name": "s1", "Query": "DEALLOCATE ALL"}
+Bind {"DestinationPortal": "p1", "PreparedStatement": "s1"}
+Describe {"ObjectType": 80, "Name": "p1"}
+Execute {"Portal": "p1"}
+Sync
+----
+
+until
+ReadyForQuery
+----
+{"Type":"ParseComplete"}
+{"Type":"BindComplete"}
+{"Type":"NoData"}
+{"Type":"CommandComplete","CommandTag":"DEALLOCATE ALL"}
+{"Type":"ReadyForQuery","TxStatus":"I"}
+
+send
+Query {"String": "DISCARD ALL"}
+----
+
+until
+ReadyForQuery
+----
+{"Type":"ParameterStatus","Name":"application_name","Value":""}
+{"Type":"ParameterStatus","Name":"TimeZone","Value":"UTC"}
+{"Type":"CommandComplete","CommandTag":"DISCARD"}
+{"Type":"ReadyForQuery","TxStatus":"I"}
+
 # Send a simple query in the middle of extended protocol, which is apparently
 # allowed. (See #41511, #33693)
 send

